components/network/vpnc/Makefile

```diff
@@ -15,21 +15,26 @@
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		vpnc
-COMPONENT_VERSION=	0.5.3
```

Comment:
so we get

oi-userland network/vpnc                                    0.5.3-2017.0.0.2:20170221T154918Z

as package version?